src/commands/workspaces.ts

```diff
@@ -108,6 +108,7 @@ export class ConnectInNewWindowCommand implements Command {
 					cancellable: true
 				},
 				async (_, cancelToken) => {
+					await this.hostService.updateSSHRemotePlatform();
```

Comment:
Let's just do it in one place in extension activation, check how I did it in next

Reply:
Force pushed to erase stupid commit history